docs/backup-recovery.md

```diff
@@ -0,0 +1,38 @@
+# Backup and Recovery
+
+## Backup and Migration Tutorial
+Because Dokku is git based, rebuilding a deployed app is as easy pushing from git. You can push to a new server by updating the dokku remote in you local app's repo.
+
+```
+$ git remote rm dokku
+$ git remote add dokku dokku@[dokku.me:dokku.me]
+$ git push dokku [master]
+```
+## Databases
+Some plugins, like the official [dokku-postgres] plugin, have built-in commands that allow non-volatile data be exported and imported. 
+
+For [dokku-postgres], use:
+```
+dokku postgres:export [db_name] > [db_name].dump
+dokku postgres:import [db_name] < [db_name].dump
+```
+
+## Volumes and Static Assets
+Dokku doesn't enforce a [300mb] limit on apps, but it's best practice to keep binary assets outside of git. Since containers are considered volatile in Dokku, external stores like s3 or volumes should be used for non-volatile items like user uploads. The [dokku-volume] plugin also supports dumping and restoring data:
+
+> volume:dump&nbsp;&nbsp;&nbsp;<app> <container_path> > filename.tar.gz&nbsp;&nbsp;&nbsp;&nbsp;Dump volume data to filename.tar.gz
+> volume:restore&nbsp;&nbsp;&nbsp;&nbsp;<app> <container_path> < filename.tar.gz&nbsp;&nbsp;&nbsp;&nbsp;Restore volume data from filename.tar.gz
```

Comment:
Open an issue. Personally I think it can be handled by docker-options, but that might be too hard for some users. Porcelain over plumbing is always nice.